event/datadeal/data_delivery_event.go

```diff
@@ -51,11 +51,7 @@ func (e DataDeliveryVoteEvent) EventHandler(event ctypes.ResultEvent) error {
 
 	voteOption, deliveredCid, err := e.verifyAndGetVoteOption(dealID, dataHash)
 	if err != nil {
-		if voteOption == oracletypes.VOTE_OPTION_UNSPECIFIED {
-			return fmt.Errorf("can't vote due to error while verify. dealID(%d). dataHash(%s): %v", dealID, dataHash, err)
-		} else {
-			log.Infof("vote NO due to error while verify. dealID(%d). dataHash(%s): %v", dealID, dataHash, err)
-		}
+		log.Infof("vote NO due to error while verify. dealID(%d). dataHash(%s): %v", dealID, dataHash, err)
```

Comment:
I think that this part should be
```go
return fmt.Errorf("can't vote due to error while verify. dealID(%d). dataHash(%s)", dealID, dataHash)
 ```
 Wdyt?

Reply:
Hmm.. I think we vote `VOTE_OPTION_NO`, not `can't vote`.
And, if it returns error, it can not make deliveryVote Tx. So, I think `log.Infof` is correct.

Reply:
Ah yes, you're right 😂